import io.netty.resolver.dns.DnsResolveContext.DnsResolveContextException;
public class FunctionUse18738 {
public void funcUse() {
DnsResolveContextException dnsresolvecontextexception = new DnsResolveContextException();
dnsresolvecontextexception.newStatic(null);
}
}